CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## v1.11.0
+
+- [core] revamp of tree search-box ui to align with vscode [#9005](https://github.com/eclipse-theia/theia/pull/9005)
+
```

Comment:
Would you also mind adding a breaking change entry, the change of the `HTMLInputElement` to `HTMLSpanElement` may have consequences to anyone that has extended the search-box class

Reply:
@kenneth-marut-work - Added a breaking change entry.